packages/tools/devtools/devtools-core/src/data-visualization/DefaultVisualizers.ts

```diff
@@ -37,17 +37,33 @@ import {
 export const visualizeSharedCell: VisualizeSharedObject = async (
 	sharedObject: ISharedObject,
 	visualizeChildData: VisualizeChildData,
-): Promise<FluidObjectTreeNode> => {
+): Promise<FluidObjectTreeNode | FluidObjectValueNode> => {
 	const sharedCell = sharedObject as SharedCell<unknown>;
 	const data = sharedCell.get();
 
 	const renderedData = await visualizeChildData(data);
 
+	const editProps = {
+		editTypes: undefined,
+	};
+
+	if (renderedData.nodeKind === VisualNodeKind.TreeNode) {
+		return {
+			...renderedData,
+			fluidObjectId: sharedCell.id,
+			children: { renderedData },
+			typeMetadata: "SharedCell",
+			nodeKind: VisualNodeKind.FluidTreeNode,
+		};
+	}
+
 	return {
+		value: "bleh",
```

Comment:
This shouldn't be needed, since we're returning a tree node. Were you getting an error without this?

Reply:
Oh, it is probably upset because `renderedData` could be an `UnknownValueNode`. We may have to handle that separately in this function.

Reply:
I get erorrs if I delete those lines, I imagined it was due to rendered data not being guaranteed to have all the necessary properties.

Reply:
Yeah. I think `visualizeChildData` can generate an UnknownObjectNode, which is incompatible. We'll have to add a separate case to support this.